src/page-multi-facility/FacilityRow.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+import Colors from "../design-system/Colors";
+import Loading from "../design-system/Loading";
+
+import CurveChartContainer from "../impact-dashboard/CurveChartContainer";
+
+import { Facility } from "./types"
+
+const markColors = {
+  exposed: Colors.green,
+  fatalities: Colors.black,
+  hospitalized: Colors.lightBlue,
+  hospitalBeds: Colors.red,
+  infectious: Colors.red,
+};
+
+interface Props {
+  facility: Facility;
+}
+
+const FacilityRow: React.FC<Props> = ({ facility }) => {
+  const {
+    name,
+    model_inputs
+  } = facility
+
+  const confirmedCases = model_inputs.confirmedCases
+
+  return (
+    <div>
+      <div className="flex flex-row h-48 mb-8 border-b border-grey-300">
+        <div className="w-2/5 flex flex-col justify-between">
+          <div className="flex flex-row">
+            <div className="w-1/4 text-red-600 font-bold">
+              {confirmedCases}
+            </div>
+            <div className="w-3/4 font-bold">
+              {name}
+            </div>
+          </div>
+          <div className="text-xs text-gray-500 pb-4 flex flex-row justify-between">
+            <div>
+              Last update: on March 25, 2020
+            </div>
+            <div className="mr-8">
+              <a className="px-1" href="#">Delete</a>
+              <a className="px-1" href="#">Edit</a>
+              <a className="px-1" href="#">Share</a>
+            </div>
+          </div>
+        </div>
+        <div className="w-3/5">
```

Comment:
generally speaking we are trying to move away from using these Tailwind css utilities in favor of `styled-components` but that's not a launch blocker IMO. depends on your appetite for writing a bunch of CSS here

Reply:
Not much of an appetite at the moment, but as an FYI, I did find a clever(?) way to use Tailwind utilities inside of styled-components which may help us ease into a full-on transition to styled components in the future:  https://github.com/Recidiviz/covid19-dashboard/pull/113/files#r409815530

Reply:
oh yeah that's cool

Reply:
Yeah that kind of refactor is definitely not release-blocking. We can file issues for tech debt tasks if we feel strongly about something.